multinode-demo/common.sh

```diff
@@ -30,7 +30,10 @@ else
       program=${BASH_REMATCH[1]}
       features="--features=cuda,erasure"
     fi
-    printf "cargo run --release --bin solana-%s %s -- " "$program" "$features"
+    if [[ -z "$DEBUG" ]]; then
+      maybe_release="--release"
+    fi
+    printf "cargo run $maybe_release --bin solana-%s %s -- " "$program" "$features"
```

Comment:
need a shellcheck directive to have it uncomplain about the lack of quotes?
you could also
```maybe_release=( --release )
...
cargo run "${maybe_release[@]}" ....
```
an empty array will not manifest on cmdline

Reply:
Actually lack of quotes was fine here.  I need a shellcheck directive to complain about superfluous quotes!